charts/gateway/README.md

```diff
@@ -1157,6 +1158,51 @@ The full default is this
     com.l7tech.server.clusterStaleNodeCleanupTimeoutSeconds=86400
     # Additional properties go here
 ```
+### DISKLESS Configuraton
+DISKLESS_CONFIG is a flag to pass gateway sensitive data to be mounted to the container gateway file system or passed as environment variables.
+By Default, true, environment variables are used to configure Gateway.
+
+When DISKLESS_CONFIG is false , Gateway will be configured from node.properties
+
+#### node.properties
+Note: 
+- Database configuration should be same as mentioned in node.properties
+```
+node.cluster.pass=newpassword
```

Comment:
while i wanted to say to use the value.yaml default sample value but i feel the need to have different pw for admin user and clusterpassphrase , in case people think they are supposed to match.